integration_tests/tests/sql/test_star_output_lower.sql

```diff
@@ -0,0 +1,7 @@
+{% set actual_output = dbt_utils.star(from=ref('data_star_uppercase_columns'), except=['field_1', 'field_2'], prefix='test_', output_lower=True ) %}
+
+{% set expected_output %}
+"field_3" as "test_field_3"
+{% endset %}
+
+{{ assert_equal_values (actual_output | trim, expected_output | trim) }}
```

Comment:
For macros that return tables (e.g. star), we tend to build a seed that is the expected shape, and a model that should make that shape, and compare the two of them. It means that we're checking the whole thing works end to end and you don't accidentally wind up with an extra comma somewhere you shouldn't etc. 

_I suspect that if we get rid of the lowercase setting on this, then this becomes a moot point as we already have tests for star and can probably get rid of the whole thing_